frontend/src/pages/MapPage/MapPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { MapContainer, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+
+const MapPage: React.FC = () => {
+    
+    const center: [number, number] = [61.494, 23.7798]; // Tampere
+    const zoom = 13;
+
+    return (
+        <div>
+            <MapContainer
+                center={center}
+                zoom={zoom}
+                scrollWheelZoom={true}
+                style={{ height: "100vh", width: "100%" }}
```

Comment:
The styles should be removed from here and put into MapPage.scss.

Reply:
Styles has been put into MapPage.scss